test/core/fixtures/transformation/validation.undeclared-variable-check/undeclared-shorthand-property/exec.js

```diff
@@ -0,0 +1 @@
+({foo})
```

Comment:
Oh shit right. Not sure, having `t.isReferenced` return `true` for a `key` seems hacky. Doubt there's a way around it though.